apps/storefront/src/pages/Dashboard/index.test.tsx

```diff
@@ -430,6 +443,7 @@ describe('when the user is associated with a company', () => {
 
     expect(beginMasquerade).toHaveBeenLastCalledWith(expect.stringContaining('companyId: 123'));
     expect(getAgentInfo).toHaveBeenLastCalledWith(expect.stringContaining('customerId: 789'));
+    expect(deleteCartReturn).toHaveBeenCalled();
```

Comment:
if we include the response only on `when`, and the test fails correctly with the wrong `id`, we don't need to include this expect here.